### ISSUE TITLE
Don't try to clone

### ISSUE THREAD

**Issue by gh-pro** [rodgomesc/vision-camera-code-scanner#51]
hi, have tested on real device ios 15.3.1 and on launch camera xcode bug and

console.

2022-03-25 09:03:39.654284+0400 Fidy[31740:7440464] [native] VisionCamera.didSetProps(_:): Updating 13 prop(s)...
2022-03-25 09:03:39.654894+0400 Fidy[31740:7441026] [native] VisionCamera.configureCaptureSession(): Configuring Session...
2022-03-25 09:03:39.654932+0400 Fidy[31740:7441026] [native] VisionCamera.configureCaptureSession(): Initializing Camera with device com.apple.avfoundation.avcapturedevice.built-in_video:0...
2022-03-25 09:03:39.655000+0400 Fidy[31740:7441026] [native] VisionCamera.configureCaptureSession(): Adding Video input...
2022-03-25 09:03:39.659604+0400 Fidy[31740:7440676] FrameProcessorBindings: Setting new frame processor...
2022-03-25 09:03:39.659643+0400 Fidy[31740:7440676] FrameProcessorBindings: Adapting Shareable value from function (conversion to worklet)...
2022-03-25 09:03:39.659872+0400 Fidy[31740:7440676] FrameProcessorBindings: Successfully created worklet!
2022-03-25 09:03:39.659944+0400 Fidy[31740:7440696] FrameProcessorBindings: Converting worklet to Objective-C callback...
Assertion failed: (object->ctx_ == ctx_ && "Don't try to clone an object backed by a different Runtime"), function cloneObject, file JSCRuntime.cpp, line 618.
dyld4 config: DYLD_LIBRARY_PATH=/usr/lib/system/introspection DYLD_INSERT_LIBRARIES=/Developer/usr/lib/libBacktraceRecording.dylib:/Developer/usr/lib/libMainThreadChecker.dylib:/Developer/Library/PrivateFrameworks/DTDDISupport.framework/libViewDebuggerSupport.dylib

**Comment by khasbilegt** [comment]
I had the same issue.

```
iOS: 15.4
react-native-vision-camera: 2.13.0
react-native-reanimated: 2.2.4
vision-camera-code-scanner: 0.16
```

**Comment by gh-pro** [comment]
@khasbilegt i solved for update react-native-reanimated for 2.3.1
